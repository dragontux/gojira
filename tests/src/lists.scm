; lists
(display '())
(newline)

(display '(a b c))
(newline)

(display '(testing this))
(newline)

(display '(foo (bar ((baz faz)) hello) cool world))
(newline)

(display (list 1 2 (+ 1 2) (list 4 5 6) 'qwerty "foo"))
(newline)
