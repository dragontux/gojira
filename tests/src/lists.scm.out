()
(a b c)
(testing this)
(foo (bar ((baz faz)) hello) cool world)
(1 2 3 (4 5 6) qwerty foo)
